examples/hello-world.rs

```diff
@@ -33,20 +33,19 @@ fn panic_handler(_info: &core::panic::PanicInfo) -> ! {
 }
 
 #[export_name = "efi_main"]
-pub extern fn main(_h: efi::Handle, st: *mut efi::SystemTable) -> efi::Status {
+pub extern "C" fn main(_h: efi::Handle, st: *mut efi::SystemTable) -> efi::Status {
     let s = [
-        0x0048u16, 0x0065u16, 0x006cu16, 0x006cu16, 0x006fu16,              // "Hello"
-        0x0020u16,                                                          // " "
-        0x0057u16, 0x006fu16, 0x0072u16, 0x006cu16, 0x0064u16,              // "World"
-        0x0021u16,                                                          // "!"
-        0x000au16,                                                          // "\n"
-        0x0000u16,                                                          // NUL
+        0x0048u16, 0x0065u16, 0x006cu16, 0x006cu16, 0x006fu16, // "Hello"
+        0x0020u16, // " "
+        0x0057u16, 0x006fu16, 0x0072u16, 0x006cu16, 0x0064u16, // "World"
+        0x0021u16, // "!"
+        0x000au16, // "\n"
+        0x0000u16, // NUL
```

Comment:
Mhhhh. This is no longer as nice to read. We could just move the whitespace into the comments, so the words still align properly. Or is that too pedantic?

Reply:
I fixed this up manually, as I suggested.